### ISSUE TITLE
having 2 buffers and closing one via 'bd' unset rnu and nu

### ISSUE THREAD

**Issue by znxuz** [JMcKiern/vim-venter#8]
open nvim and have more than 2 buffers, then close one via `:bd` while in venter mode unset relative number and number.

using neovim: NVIM v0.6.0-dev+206-g284199bc4

**Comment by JMcKiern** [comment]
Hmm, I can't replicate this with NVIM v0.5.0 (their latest nightly job failed so there is no nightly release is available - and I can't be bothered to build from source).

The only time `number` and `relativenumber` are modified is in the snippet below but this should only change what is shown in the newly created window from L135 as both settings are "local to window".

https://github.com/JMcKiern/vim-venter/blob/819fe9dcb81ca2127d58b2775ba1cd19b9da1472/autoload/venter.vim#L135-L143

If I remember to, I will try to replicate this bug with the next neovim nightly.

**Comment by znxuz** [comment]
Maybe it is caused by other plugins so I tested again with a minimal config(config.vim) consisting of
```
set rnu nu

call plug#begin('$XDG_DATA_HOME/nvim/site/plugged')
	Plug 'jmckiern/vim-venter'
call plug#end()
nnoremap ,z <CMD>VenterToggle<CR>
```
and running `nvim -u config.vim buf1 buf2`, using `,z` start venter, and then just `:bd` to delete the first buffer and untoggle venter mode, displaying a `[No Name]` buffer with relative number and number unset, which can be resetted via `set rnu nu` again.
Removed the nightly and installed the stable version from official repo (pacman -S neovim) caused the same thing on my machine.
Unfortunately I am so illiterate with it comes to vimL, and it's a minor thing anyway and I've been using a custom command to delete buffers while in venter mode via `b# | bd #`, which works flawlessly except when there is only a single buffer.
